packages/cursorless-vscode/src/ide/vscode/VscodeFileSystem.ts

```diff
@@ -1,26 +1,87 @@
-import { Disposable, FileSystem, PathChangeListener } from "@cursorless/common";
-import { RelativePattern, workspace } from "vscode";
-import * as path from "path";
+import {
+  Disposable,
+  FileSystem,
+  PathChangeListener,
+  RunMode,
+} from "@cursorless/common";
+import { isAbsolute } from "path";
+import { TextDecoder } from "util";
```

Comment:
I'm pretty sure this is a node import, no?

Reply:
update: looks like this import isn't necessary, as `TextDecoder` is globally available in both node and browser